distribution/changelog.txt

```diff
@@ -92,6 +92,7 @@
 - Fix: [#13130] Android always defaulting to UK locale for language, currency and temperature.
 - Fix: [#13397] Ride simulation causes strange station behaviour and makes the ride unusable in some cases.
 - Fix: [#16791] Rotodrop top piece replicates when walls are placed around it and clearance checks are disabled (original bug).
+- Fix: [#17774] Misplaced/missing land and construction rights tiles in RCT1 & RCT2 scenarios.
```

Comment:
Almost there, this has to be moved to 0.4.7.

I think we also need to bump the network version, since people will have these maps behaving differently